pinot-common/src/main/proto/plan.proto

```diff
@@ -76,8 +76,9 @@ message LiteralField {
     bool boolField = 1;
     int32 intField = 2;
     int64 longField = 3;
-    double doubleField = 4;
-    string stringField = 5;
+    float floatField = 4;
```

Comment:
The code needs to be regenerated (hopefully it doesn't complain of reordering ?) but I guess it is gitignored

Reply:
correct. this will be regenerated.